code/modules/bitrunning/server/obj_generation.dm

```diff
@@ -74,6 +79,7 @@
 			new /obj/item/flashlight,
 		)
 
+	if(include_loadout)	avatar.equip_outfit_and_loadout(new /datum/outfit(), prefs) // BUBBER EDIT - LOADOUTS
```

Comment:
```suggestion
	if(include_loadout)
		avatar.equip_outfit_and_loadout(new /datum/outfit(), prefs) // BUBBER EDIT - LOADOUTS
```
Does DM allow this sort of syntax? The inline if specifically. In any case I'd break this up over two lines, we do every if like that, no reason not to be consistent here imo.